### ISSUE TITLE
Attributes "copts" seems not working

### ISSUE THREAD

**Issue by wudisheng** [bazelbuild/rules_foreign_cc#821]
For example, in a clean workspace environment without system and user rc files, `@openssl//:openssl` in the following case compiles, and there seems no sign of `-flto` or the definitely invalid `-asdfghjk` in `bazel-bin/external/openssl/openssl_foreign_cc/Configure.log` or `bazel-bin/external/openssl/openssl_foreign_cc/build_script.sh`.

However, a workaround would be using `env = {"CFLAGS": "...", "CXXFLAGS": "..."}`, which actually appends to the toolchain's flags (instead of replacing, just like what I expected from `copts`), so I guess the priority could be lowered.

Attaching a minimal reproducible setup, I'm using Bazel 4.2.1 and rules_foreign_cc 0.6.0 currently.
```
workspace(name = "bug")

load(
    "@bazel_tools//tools/build_defs/repo:http.bzl",
    "http_archive",
)
http_archive(
    name = "rules_foreign_cc",
    strip_prefix = "rules_foreign_cc-0.6.0",
    url = "https://github.com/bazelbuild/rules_foreign_cc/archive/0.6.0.tar.gz",
)
http_archive(
    name = "openssl",
    sha256 = "281e4f13142b53657bd154481e18195b2d477572fdffa8ed1065f73ef5a19777",
    strip_prefix = "openssl-OpenSSL_1_1_1g",
    urls = ["https://github.com/openssl/openssl/archive/OpenSSL_1_1_1g.tar.gz"],
    build_file_content = """
load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
package(default_visibility = ["//visibility:public"])
filegroup(name = "source", srcs = glob(["**"]))
configure_make(
    name = "openssl",
    configure_command = "config",
    configure_options = ["no-zlib", "no-shared", "enable-cms"],
    out_static_libs = ["libcrypto.a", "libssl.a"],
    lib_source = ":source",
    configure_in_place = True,
    targets = ["", "install_dev"],
    args = ["-j$(nproc)"],
    copts = ["-Werror", "-asdfghjk"],
)
""",
)

load(
    "@rules_foreign_cc//foreign_cc:repositories.bzl",
    "rules_foreign_cc_dependencies",
)

rules_foreign_cc_dependencies()
```

**Comment by jsharpe** [comment]
A fix for this went into https://github.com/bazelbuild/rules_foreign_cc/commit/d93bd96dc719760c968b54730258ad0a5b10f8fb but its not made it into a release yet.

**Comment by jsharpe** [comment]
This should be fixed in release 0.7.0
